site/content/docs/5.3/examples/download-app/index.html

```diff
@@ -0,0 +1,373 @@
+          </svg>
+        </svg>
+      </div>
+    </div>
+  </div>
+
+  <div class="py-4 py-md-5 bg-supporting-yellow">
+    <div class="container-xxl d-flex justify-content-center pt-md-2 pb-2">
+      <div class="col-12 col-md-6">
+        <h2 class="h1 mb-2 mb-md-3 text-center">Your feedback is valued</h2>
+        <h3 class="mb-md-3 text-center">Let us know if you spot anything to correct or improve</h3>
```

Comment:
Other subtitles in this page are not `<h3>`s. Is this one particular?

Reply:
There is one under the `A showcase mobile application`. I can remove both if you think it's better without. It doesn't bring much information to have them imo.

Reply:
I'd say both are not really titles/not really sub-sections so would be prolly safe to remove them.

Reply:
Removed